tools/SetupFlow/DevHome.SetupFlow/Models/CloneRepoTask.cs

```diff
@@ -280,10 +280,16 @@ IAsyncOperation<TaskFinishedState> ISetupTask.Execute()
                 }
             }
 
-            // TODO: Is this the best place to add the repository to the database?

```

Comment:
Change Looks fine, but I wonder if we should make a single issue and you can then put all your TODOs in there. and then reference that single issue in the code, since all of them will be resolved before release

Reply:
I'll look into that.  I add more TODO's as I code to remind myself of future (pre-production) tasks to finish.  I'll look into another way to record my thoughts and not have TODO's everywhere.